usr/lib/xtable/Makefile

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: (c) 2024 Mohammad Shehar Yaar Tausif <sheharyaar48@gmail.com>
+# SPDX-License-Identifier: MIT OR GPL-2.0-only
+
+LUAXTABLE_MODULE ?= test
+
+CFLAGS = -DLUAXTABLE_MODULE=\"${LUAXTABLE_MODULE}\" -O2 -Wall -I../../../lib
+XTABLES_SO_DIR = $(shell pkg-config xtables --variable xtlibdir)
+LUA_FLAGS = -llua -lm
+
+all:
+	make libxt_${LUAXTABLE_MODULE}.so
+
+install:
+	sudo cp libxt_${LUAXTABLE_MODULE}.so  ${XTABLES_SO_DIR}
```

Comment:
I wouldn't require LUAXTABLE_MODULE to be defined on `install`, I would just copy `libxt_*.so`